tools/install.pl

```diff
@@ -123,8 +124,19 @@
     );
     say("\r\nInstalling Perl modules... This might take a while.\r\n");
 
-    if ( system("cpanm --installdeps ./tools/. --notest") != 0 ) {
-        die "Something went wrong while installing Perl modules - Bailing out.";
+    # libarchive is not provided by default on macOS, so we set the correct env vars
```

Comment:
I suppose you installed it using homebrew? Would be good to add that detail either here or in the documentation.

Reply:
Yes, I did. I’m actually working on a formula for LANraragi since the beginning :)
As for manual installation, using Homebrew is obviously also the best/fastest way of doing it.
I had to install the following: `cpanminus libarchive libjpeg libpng node openssl perl redis`